pkgs/applications/audio/ardour/default.nix

```diff
@@ -126,7 +121,7 @@ stdenv.mkDerivation rec {
     suil
     taglib
     vamp-plugin-sdk
-  ] ++ lib.optionals videoSupport [ harvid xjadeo ];
```

Comment:
Does this PR also remove video support entirely? Or does the amended `postPatch` script supersede this somehow?

Reply:
When testing this locally with some existing soundtracking work it looks like the video timeline no longer works out of the box. Would you mind commenting on why these changes were reverted so that I can have a chance at addressing this?

Reply:
There's a very simple explanation:  I messed up!
I'm building the improved version now, and will push it after testing.

Reply:
I did not test the video stuff, since I never used it.